mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt

```diff
@@ -40,13 +54,20 @@ class MPSLanguageRepository(private val repository: SRepository) : ILanguageRepo
         return MPSConcept(MetaAdapterFactory.getAbstractConcept(conceptDescriptor))
     }
 
-    override fun getAllConcepts(): List<IConcept> {
-        val result = mutableListOf<IConcept>()
-        LanguageRegistry.getInstance(repository).withAvailableLanguages { language ->
-            result.addAll(language.identity.concepts.map { MPSConcept(it) }.toList())
-        }
-        return result
-    }
+    fun getConcept(uid: String): SAbstractConcept? = concepts[uid]?.concept
```

Comment:
Why not `resolveConcept(uid)?.concept`?

Reply:
Good point! I've missed that method for some reasons. I applied your suggestion and now we do not need the cache anymore. Thank you for the suggestion again.